screens/CreateAccScreen.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import {
+  Image,
+  Platform,
+  ScrollView,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+  TouchableHighlight
+} from 'react-native';
+import { WebBrowser } from 'expo';
+import HomeScreen from './HomeScreen';
+import { MonoText } from '../components/StyledText';
+import { InputText } from '../components/InputText';
+
+
+export default class IntroAppScreen extends React.Component {
+  static navigationOptions = {
+    header: null,
+  };
+
+  render() {
+    return (    	
+      <View style={styles.container}>
+        <View style={[{ position: 'absolute', top: 0 }]}>
+          <Image
+            source={
+              __DEV__
+                ? require('../assets/images/logo.png')
+                : require('../assets/images/logo.png')
+            }
+            style={styles.welcomeImage}
+          />
+        </View>
+
+        <Text style={styles.welcomeText}>Let's Get Started</Text>
+        <InputText placeholder="Email Address"/>
+        <InputText placeholder="Confirm Email Address"/>
+        <InputText placeholder="Password" secureTextEntry={true}/>
+
+        <Text style={styles.text}>Accept Seller Agreement and Privacy Policy</Text>
+        <Text style={styles.textSmall}>View Seller Agreement and Privacy Policy</Text>
+        
+        <View style={[{ width: "100%", backgroundColor: "#F39F86", position: 'absolute', bottom: 0, borderRadius: 4, color: "red!important" }]}>
+  				<TouchableHighlight onPress={() => this.props.navigation.navigate('Home')}>
+          	<Text style = {styles.gsText}>
+            	Sign Upp
+            </Text>
+        	</TouchableHighlight>
+				</View>	   
+
+      </View>    
+    );
+  }
+}
+
+const styles = StyleSheet.create({
```

Comment:
Just curious if the styles and look & feel settings should be declared in a separate file so it can be applied across all the screens? Please advise.

Reply:
created separated styles.js and use it by import from screens. style in RN is not css file but separating is not bad.